packages/cardano-services/src/Http/openApi.json

```diff
@@ -0,0 +1,106 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Server Metadata",
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/meta": {
+      "post": {
+        "summary": "HTTP Server metadata POST",
+        "operationId": "server-metadata-post",
+        "responses": {
+          "200": {
+            "description": "HTTP Server metadata",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ServerMetadata"
+                }
+              }
+            }
+          },
+          "400": {
```

Comment:
Does it ever respond with 400? There are no arguments to this request :thinking:

Reply:
RIght, there is no such case since we validate the build info on startup, will remove it from open API schema @mkazlauskas